src/mpi/coll/op/opland.c

```diff
@@ -64,7 +64,7 @@ void MPIR_LAND(void *invec, void *inoutvec, int *Len, MPI_Datatype * type)
                 /* We previously supported floating point types, although I question
                  * their utility in logical boolean ops [goodell@ 2009-03-16] */
                 MPIR_OP_TYPE_GROUP(FLOATING_POINT)
-                MPIR_OP_TYPE_GROUP(FLOATING_POINT_EXTRA)
+                /* MPIR_OP_TYPE_GROUP(FLOATING_POINT_EXTRA) */
```

Comment:
What happens if we leave these uncommented? Regardless, I think we probably want to move this and the other one to a separate commit. I think a reader of this series of commits would be surprised to know this change was included.